Master-OB-OpenAPI.json

```diff
@@ -12661,6 +12664,33 @@
         "items": {
           "$ref": "#/components/schemas/Node"
         }
+      },
+      "ProdBreaker": {
+        "type": "object",
+        "description": "Circuit Breaker",
+        "properties": {
+          "CurrentACMax": {
+            "$ref": "#/components/schemas/CurrentACMax"
+          },
+          "CurrentFaultSupplyMax": {
+            "$ref": "#/components/schemas/CurrentFaultSupplyMax"
+          }
+        }
+      },
+      "CurrentFaultSupplyMax": {
```

Comment:
Curious why call it CurrentFaultSupplyMax and not CurrentFaultRating?

Reply:
Because equipment such as Energy Storage Systems can have both supply and output fault current.

Reply:
So this is for either Fault or Supply current, right? I was confused by thinking it was "fault supply" as if it was the maximum current that could be supplied to a fault.